qutebrowser/config/configtypes.py

```diff
@@ -305,6 +313,9 @@ class FlagList(List):
 
     combinable_values = None
 
+    def __init__(self, none_ok=False, valid_values=None):
+        super().__init__(none_ok, valid_values)
+
```

Comment:
I see, it evolved like this since I realized "oh FlagList needs that parameter upon construction" and then "oh then List needs it too" ;) Adapting.